ci/checks/style.sh

```diff
@@ -54,21 +54,8 @@ HEADER_META=`ci/checks/headers_test.sh`
 HEADER_META_RETVAL=$?
```

Comment:
Just noticed that, `HEADER_META_RETVAL` is not used in `RETVALS` list for determining the style check failure. 
Is this an oversight or intentional?

Reply:
I think this is a bug. @vyasr Can you confirm if that was an unintentional change in https://github.com/rapidsai/cudf/commit/1ab315a1f019d1223c19b9b89ee116e0024aa254?

Reply:
Yes, this is definitely a bug on my part.